.vscode/settings.json

```diff
@@ -15,5 +15,27 @@
   "go.formatTool": "gofmt",
   "go.formatFlags": [
     "-s"
-  ]
+  ],
+  "workbench.colorCustomizations": {
```

Comment:
this needs to be removed, it's a bit too much :-)

Reply:
Sorry, this is from the Peacock extension. I will remove it.

Reply:
It was pretty colorful, I admit 😄